.kokoro/continuous/readme.cfg

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Google LLC
```

Comment:
New files should have 2021 year.

Reply:
This is just a copy of what the Kokoro config file looks like now